Makefile

```diff
@@ -10,13 +10,6 @@ services: args?=up -d --wait
 services: python
 	@docker compose $(args)
 
-.PHONY: db
-$(call help,make db,initialize the DB and upgrade it to the latest migration)
-db: args?=upgrade head
-db: python
-	@tox -qe dev --run-command 'python bin/make_db'
-	@tox -qe dev  --run-command 'alembic $(args)'
```

Comment:
This is expected: the cookiecutter removed `make db` from non-Pyramid projects (it wasn't working anyway: there's no `bin/make_db` in `data-tasks`)